### PR TITLE
Add link to composite databases introduction section

### PR DESCRIPTION
**Pull request by Hunterness**
Fix leftover part of https://github.com/neo4j/neo4j-documentation/pull/1609#discussion_r977507795, adding link introduced in https://github.com/neo-technology/neo4j-manual-modeling/pull/3029

### CODE REVIEW

**Review by mnd999 — APPROVED** [review]
I built it, and I think the link looks okay. There seem to be some issues with crosslinks in my build but it seems equally broken as the link above so I think it's fine.

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala [hunk 1]
```diff
@@ -159,9 +159,9 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
         }
       }
       section("Creating composite databases", "administration-databases-create-composite-database", "enterprise-edition") {
-        // TODO: add a link to the place explaining what composite databases are
         p(
-          """Composite databases do not contain data, but they reference to other databases that can be queried together through their constituent aliases.""".stripMargin)
+          """Composite databases do not contain data, but they reference to other databases that can be queried together through their constituent aliases.
+            |For more information about composite databases, see <<operations-manual#composite-databases-introduction, Composite database introduction>>.""".stripMargin)
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala]:
```suggestion
            |For more information about composite databases, see <<operations-manual#composite-databases-introduction, Operations Manual -> Composite database introduction>>.""".stripMargin)
```

**Review by AlexicaWright — COMMENTED** [review]
We'd like to highlight when we link inter-manual.

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala (hunk 1 above)
Comment (mnd999) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala]: '→' surely?

Comment (AlexicaWright) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala]: Looks fancier than -> , but ->  is what is used elsewhere, so unless you want to update all the other arrows throughout the manual, I think this should be -> .

**Review by AlexicaWright — APPROVED** [review]
👍 LGTM
Thank you @Hunterness !

**Pull request merged**